src/commands/database/mysql/user/update.tsx

```diff
@@ -0,0 +1,126 @@
+import { ExecRenderBaseCommand } from "../../../../lib/basecommands/ExecRenderBaseCommand.js";
+import { Flags } from "@oclif/core";
+import { ReactNode } from "react";
+import {
+  makeProcessRenderer,
+  processFlags,
+} from "../../../../rendering/process/process_flags.js";
+import { Success } from "../../../../rendering/react/components/Success.js";
+import assertSuccess from "../../../../lib/apiutil/assert_success.js";
+import type { MittwaldAPIV2Client } from "@mittwald/api-client";
+type UpdateResult = void;
+type MyQSLUserUpdateData = Parameters<
+  MittwaldAPIV2Client["database"]["updateMysqlUser"]
+>[0]["data"];
+
+export default class Update extends ExecRenderBaseCommand<
+  typeof Update,
+  UpdateResult
+> {
+  static description = "Create a new mysql user";
+  static flags = {
+    ...processFlags,
+    "mysql-user-id": Flags.string({
+      required: true,
+      description: "MySQL User ID of the user to be updated",
+    }),
+    "access-level": Flags.string({
+      description: "Access level for this MySQL user",
+      options: ["readonly", "full"],
+    }),
```

Comment:
You can strictly type these flags by using `Flags.custom<"full" | "readonly">({...})()`. See `database mysql compare` for comparison. You can also use `Flags.custom` to define a common flag definition (somewhere in `lib/resources/database/mysql`) and reuse that flag definition in all required places.